civihr_default_theme/scss/partials/_button.scss

```diff
@@ -0,0 +1,16 @@
+.btn-default {
+  border-color: $gray;
+  color: $gray;
+
+  &.active,
+  &.active:hover {
+    border-color: $gray;
+    background: $gray;
+    color: $light;
```

Comment:
Is it really necessary to redefine all these properties? I tried on my local to have a `.btn.btn-default.active` without those properties and it doesn't seem to look broken